mobile_config/src/mobile_radio_tracker.rs

```diff
@@ -1,10 +1,15 @@
-use std::{collections::HashMap, time::Duration};
+use std::{collections::HashMap, sync::Arc, time::Duration};
 
 use chrono::{DateTime, Utc};
 use futures::{Stream, StreamExt, TryFutureExt, TryStreamExt};
+use helium_crypto::PublicKeyBinary;
+use sqlx::Row;
 use sqlx::{Pool, Postgres, QueryBuilder};
+use std::str::FromStr;
 use task_manager::ManagedTask;
+use tokio::sync::RwLock;
 
+pub type TrackedRadiosMap = HashMap<PublicKeyBinary, DateTime<Utc>>;
```

Comment:
I'm wondering if this would make sense as it's own type
```rs
#[derive(Clone)]
pub struct TrackedRadiosMap(Arc<RwLock<HashMap<PublicKeyBinary, DateTime<Utc>>>);
```

We would get Clone for free.
And we could hang methods like `get_updated_radios` off of it, and it would self update.
And other access methods could manage the read locking instead of the user.

Just thinking out loud.

Reply:
Is it want you mean?
https://github.com/helium/oracles/commit/48dcfd44a4d32d7673c4536a4ba8ac7f1d32c45d